src/solvers/dg/indicators.jl

```diff
@@ -63,14 +63,16 @@ function Base.show(io::IO, indicator::IndicatorHennemannGassner)
 end
 
 function Base.show(io::IO, ::MIME"text/plain", indicator::IndicatorHennemannGassner)
-  println(io, "IndicatorHennemannGassner(")
-  println(io, "- ", indicator.variable)
-  println(io, "- alpha_max:    ", indicator.alpha_max)
-  println(io, "- alpha_min:    ", indicator.alpha_min)
-  println(io, "- alpha_smooth: ", indicator.alpha_smooth)
-  print(io,   "- cache with fields:")
-  for key in keys(indicator.cache)
-    print(io, " ", key)
+  if get(io, :compact, false)
+    show(io, indicator)
+  else
+    setup = [
+             "indicator variable" => indicator.variable,
+             "max. α" => indicator.alpha_max,
+             "min. α" => indicator.alpha_min,
+             "α smoothing" => (indicator.alpha_smooth ? "yes" : "no"),
```

Comment:
```suggestion
             "smooth α" => (indicator.alpha_smooth ? "yes" : "no"),
```
To be consistent with `save initial solution`, `save final solution`, ``save analysis to file etc.? There, you ask questions (in an abbreviated form without question mark) and the answer is either `yes` or `no`.

Reply:
Can't believe this happened to *me*, of all people. These inconsistencies everywhere... :scream: 

Thanks!